### PR TITLE
Dune, opam 2 and OCaml 4.07.1

### PR DESCRIPTION
**Pull request by dra27**

### CODE REVIEW

File: README.org [hunk 1]
```diff
@@ -9,7 +9,7 @@
    - macOS 10.13.2
```
Comment (avsm) [inline README.org]:
```suggestion
   - macOS 10.13.6
```

File: README.org [hunk 2]
```diff
@@ -9,7 +9,7 @@
...
    - Debian 9.3
    - Fedora 27
```
Comment (avsm) [inline README.org]:
```suggestion
   - Fedora 29
```

File: README.org [hunk 3]
```diff
@@ -42,7 +50,7 @@
     (just put it somewhere in your PATH with the name ~opam~):
 
     #+BEGIN_SRC bash
-    wget https://github.com/ocaml/opam/releases/download/1.2.2/opam-1.2.2-x86_64-Darwin
+    wget https://github.com/ocaml/opam/releases/download/2.0.3/opam-2.0.3-x86_64-darwin
```
Comment (avsm) [inline README.org:64]: 2.0.4 or too soon?

**Review by avsm — CHANGES_REQUESTED** [review]

File: README.org (hunk 3 above)
Comment (dra27) [reply README.org:64]: It's not there yet, but can go in this PR!

**Pull request merged**